ocs_ci/helpers/dr_helpers.py

```
@@ -1687,71 +1687,163 @@ def validate_vgrc_count():
+                logger.info(
+                    "The value of lastGroupSyncTime in drpc is not updated. Retrying..."
+                )
+    except TimeoutExpiredError:
+        if wait_for_set:
+            error_msg = "lastGroupSyncTime was not set within 3 scheduling intervals"
+        else:
+            error_msg = (
+                f"lastGroupSyncTime did not change from "
+                f"{previous_last_group_sync_time} within 3 scheduling intervals"
+            )
+        logger.error(error_msg)
```

Ruff (0.15.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)